### PR TITLE
[SELS-TASK] Setup React Project

### PR DESCRIPTION
**Pull request by kenorenzsun**
[Asana Link]
[https://app.asana.com/0/1203369406376494/1203389366920492/f](url)
[Commands]

npm install
[Pre-conditions]
None
[Expected Output]

React environment can now use API connected to laravel project
React project now have routes for protected navigation 
[Notes]
None

### CODE REVIEW

File: els-front/src/components/ProtectedRoute.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import {
+    Navigate,
+  } from 'react-router-dom';
+  
+const ProtectedRoute = ({ user, redirectPath = '/',for_auth = false,children }) => {
```
Comment (chevyB) [inline els-front/src/components/ProtectedRoute.js]:
```suggestion
const ProtectedRoute = ({ user, redirectPath = '/', for_auth = false, children }) => {
```

- spaces
- add new line at EOF, also apply to other files

File: els-front/src/App.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
...
+import { BrowserRouter} from "react-router-dom"
+import DefaultRoutes from "./plugins/Router/index"
 
 function App() {
   return (
-    <h1 className="text-3xl font-bold underline">
-      Hello world!
-    </h1>
+      <BrowserRouter>
+        <DefaultRoutes></DefaultRoutes>
```
Comment (chevyB) [inline els-front/src/App.js]:
```suggestion
        <DefaultRoutes />
```

File: els-front/src/plugins/Router/index.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            path: "/",
+            element: 
+                <ProtectedRoute user={true} redirectPath="/auth/login"> 
+                    <Main />
+                </ProtectedRoute>
+        },
+        { 
+            path: "/auth", 
+            element: 
+                <ProtectedRoute user={true} for_auth redirectPath="/s">
```
Comment (chevyB) [inline els-front/src/plugins/Router/index.js]: is `/s` intentional?

Comment (kenorenzsun) [reply els-front/src/plugins/Router/index.js]: Ah yes sir I forgot to change it back

**Pull request merged**